src/platforms/accelo/client.ts

```diff
@@ -0,0 +1,20 @@
+import { formatUrl, makeRequestFactory } from '@/sdk/client';
+
+const request = makeRequestFactory(async (auth, options) => {
+  const { answers } = await auth.getMetadata();
+  return {
+    ...options,
+    url: formatUrl(
+      `https://${answers.subdomain}.api.accelo.com/api/v0`,
```

Comment:
This is a separate question. Should we not enforce API version? I know we've done it for some of our existing integrations, but I'm wondering if that's just too restrictive

Perhaps we leave that to the end user.

Reply:
I think it can go either way - for some it's a matter of convenience, for others (like with hubspot) there might be some real reason to use an old API. 

Either way preventing this from coming through in these PRs will probably not be possible so if we want this we'd need to go and manually undo it.